### DISCUSSION TITLE
Mouse input is "ignored" in certain areas if used after a missed click or a drag

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#18797] [Q&A] [answered]
When the user misses the object they wanted to click (which could be a button, a dropdown, or whatever), there seems to be this small amount of time where the next click is not "detected" (whether the mouse is instantly released or held down).
Here are some examples I found:

https://user-images.githubusercontent.com/43073074/174835317-b2d2fc4c-8ad6-4a61-a9d4-8de99d15d5dc.mp4

https://user-images.githubusercontent.com/43073074/174838124-7ded348d-e83a-48a9-9a98-97bc1997f688.mp4

https://user-images.githubusercontent.com/43073074/174838149-b9e794e4-496b-42bc-a37e-284d0064fa98.mp4

**Comment by peppy** [comment, marked as answer]
This is the "double click" logic failing badly. Needs to be aborted/reset when the first and second click arrive at different drawables.

**Reply by frenzibyte** [reply]
Tracking via https://github.com/ppy/osu/issues/18838
